yt_dlp/extractor/opencast.py

```diff
@@ -134,10 +137,12 @@ def _real_extract(self, url):
 
 
 class OpencastPlaylistIE(OpencastBaseIE):
-    _VALID_URL = r'''(?x)
-                            https?://(?P<host>%s)/engage/ui/index.html\?.*?
-                            epFrom=(?P<id>%s)
-                    ''' % (OpencastBaseIE._INSTANCES_RE, OpencastBaseIE._UUID_RE)
+    _VALID_URL = rf'''(?x)
+                            https?://(?P<host>{OpencastBaseIE._INSTANCES_RE})
+                            (/engage/ui/index.html\?(?:[^#]+&)?epFrom=|
+                            /ltitools/index.html\?(?:[^#]+&)?series=)
+                            (?P<id>{OpencastBaseIE._UUID_RE})
+                    '''
```

Comment:
```suggestion
    _VALID_URL = rf'''(?x)
        https?://(?P<host>{OpencastBaseIE._INSTANCES_RE})(
            /engage/ui/index.html\?(?:[^#]+&)?epFrom=|
            /lttools/index.html\?(?:[^#]+&)?series=
        )(?P<id>{OpencastBaseIE._UUID_RE})'''
```

Reply:
I do not think this will work. That is the same change as you suggested before. I think the regex struggles with the newline

Reply:
yep I tested it, the tests fail

Reply:
just a typo